query/language/define.feature

```diff
@@ -69,7 +68,24 @@ Feature: TypeQL Define Query
       | label:child  |
 
 
-  Scenario: when defining that a type owns a non-existent thing, an error is thrown
+  Scenario: defining an entity type sub when a different sub is already defined errors
+    Given typeql schema query
+      """
+      define
+      person sub parent;
+      entity parent @abstract;
+      entity not-parent;
+      """
+    Given transaction commits
+
+    When connection open schema transaction for database: typedb
+    Then typeql schema query; fails with a message containing: "different 'person sub parent' is already defined"
```

Comment:
Started using `fails with a message containing` which I had introduced in `driver.feature` when I am interested in a reason of the fail.